libs.autobuild

```diff
@@ -329,7 +330,9 @@ in_flavor 'master', 'stable' do
     metapackage 'external/freenect', 'drivers/freenect'
 
 
-    cmake_package 'control/urdfdom'
+    cmake_package 'control/urdfdom' do |pkg| 
+        pkg.define "BUILD_TESTING", "OFF"
+    end
```

Comment:
N.B.: Ubuntu does provide `liburdfdom-dev` (and `liburdfdom-headers-dev`), which could make this actually obsolete.
However, on Ubuntu 20.04 it seems to be only version 1.0.4, Ubuntu 22.04 has version 3.0.1, Ubuntu 24.04 has 4.0.0 (not sure what version is required).

Reply:
It is still not obsolete, as someone could add an osdep override....